src/react/JSONTree/obj-type.js

```diff
@@ -1,3 +1,7 @@
 export default function(obj) {
-  return Object.prototype.toString.call(obj).slice(8, -1);
+  const type = Object.prototype.toString.call(obj).slice(8, -1);
+  if (type === 'Object' && typeof obj[Symbol.iterator] === 'function') {
```

Comment:
When using the babel `Map` polyfill, the `.toString()` returns 'Map'.

Change to
```
  if ((type === 'Object' || type === 'Map') && typeof obj[Symbol.iterator] === 'function') {
```